packages/connector-jsdom/README.md

```diff
@@ -63,6 +48,13 @@ The set of settings supported by jsdom connector are:
 }
 ```
 
+## Known issues
```

Comment:
We can probably remove this. I don't think this has applied since we switched to using a snapshot approach for DOM traversal.